Resources/public/js/apps/ez-platformuiapp.js

```diff
@@ -266,7 +234,6 @@ YUI.add('ez-platformuiapp', function (Y) {
          * @protected
          */
         _retryAction: function (retryAction) {
```

Comment:
AFAIK, `_retryAction` and `close` are not used anywhere now, so they can safely be removed.